modules/installation-user-infra-machines-advanced-customizing-live-iso.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_{context}"]
+= Customizing a live {op-system} ISO image
+You can customize a live {op-system} ISO image directly with the `coreos-installer iso customize` subcommand. When the ISO image boots, the customizations are applied automatically.
```

Comment:
Maybe after this sentence another sentence like:

> This can be used to configure the ISO to automatically install {op-system}.

This sets up the reader for the procedure a bit more.